client/state/plans/actions.js

```diff
@@ -54,28 +44,10 @@ export const plansRequestFailureAction = error => {
 };
 
 /**
- * Fetch WordPress.com plans
+ * Action creator to request WordPress.com plans: REQUEST
  *
- * @return {Promise} Promise
+ * @return {Object} action object
  */
-export const requestPlans = () => {
-	return dispatch => {
-		dispatch( plansRequestAction() );
-
-		return wpcom
-			.withLocale()
-			.plans()
-			.list( { apiVersion: '1.4' } )
-			.then( data => {
-				dispatch( plansRequestSuccessAction() );
-				dispatch( plansReceiveAction( data ) );
-			} )
-			.catch( ( error ) => {
-				const message = error instanceof Error
-					? error.message
-					: error;
-
-				dispatch( plansRequestFailureAction( message ) );
-			} );
-	};
-};
+export const requestPlans = () => ( {
+	type: PLANS_REQUEST
```

Comment:
I think this is already the convention, no? or maybe it's `_FETCH`. I prefer `_REQUEST` because that indicates the _need_ of data more than the _act_ of fetching it. In the end I decided it would be better to intercept existing actions than creating a new nomenclature for them.